azkaban-soloserver/src/package/conf/azkaban.properties

```diff
@@ -24,6 +24,7 @@ h2.create.tables=true
 velocity.dev.mode=false
 
 # Azkaban Jetty server properties. Ignored in tomcat
+jetty.hostname=localhost
```

Comment:
Could you update the comment to say that even on tomcat the jetty.* properties are used to generate links for email notifications?